docker/scripts/setup_env

```diff
@@ -85,10 +97,10 @@ esac
 bundle install --without test development --path vendor/bundle
 
 # Configure the unicorn server
-cp -a config/unicorn.rb.example config/unicorn.rb
-sed -ri 's/^listen .*$/listen ENV["PORT"]/' config/unicorn.rb
-sed -ri 's/^stderr_path.*$//' config/unicorn.rb
-sed -ri 's/^stdout_path.*$//' config/unicorn.rb
-sed -ri 's/^pid.*$//' config/unicorn.rb
+sed -r config/unicorn.rb.example \
+    -e 's/^ *listen .+/listen ENV["DOMAIN"] || "127.0.0.1:3000"/' \
```

Comment:
I think that should stay `PORT`, `DOMAIN` is supposed to be the external URL of the Huginn server like `https://youdomain.com` and is used in the application for redirects. I don't think there is a use case in which the unicorn server inside docker should listen do anything but localhost.

Reply:
My mistake: I should have written `listen $IP:$PORT` (to achieve similar behavior as in the multi-process image). Localhost may not be sufficient for all use cases. For example, one can add an nginx reverse proxy to one of the `docker-compose.yml` files. The inter-container communication would not be on the loopback interface.